### PR TITLE
Document materialized_only option

### PR DESCRIPTION
**Pull request by svenklemm**

### CODE REVIEW

**Comment by solugebefola** [comment]
merge?

**Comment by svenklemm** [comment]
This is not for 1.6.1 and since the PR adding this is not yet merged I didnt merge this one either

**Comment by solugebefola** [comment]
Do you mean that the PR for timescale/timescaledb is not yet merged?

**Comment by svenklemm** [comment]
Yes this depends on https://github.com/timescale/timescaledb/pull/1685 which will probably be merged shortly after 1.6.1 has been released

File: api.md [hunk 1]
```diff
@@ -1186,6 +1186,11 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
 | Refresh interval controls how often the background materializer is run. Note that if `refresh_lag` is set to `-<bucket_width>`, the continuous aggregate will run whenever new data is received, regardless of what the `refresh_interval` value is. | `INTERVAL`|By default, this is set to twice the bucket width (if the datatype of the bucket_width argument from the `time_bucket` expression is an `INTERVAL`), otherwise it is set to 12 hours.|
 |   |   |   |
 |**Name**|||
+|`timescaledb.materialized_only`|||
+|**Description**|**Type**|**Default**|
+| Combine data from the materialization hypertable with data from the raw hypertable when querying the continuous aggregate view. | `BOOLEAN` | false |
```
Comment (mkindahl) [inline api.md]: It is not clear if the combine happens when the option is true or false. I would suggest to write it out explicitly.

**Pull request merged**